vdom/apply-properties.js

```diff
@@ -52,9 +52,9 @@ function removeProperty(node, propName, propValue, previous) {
         node.style[i] = "";
       }
     } else if (typeof previousValue === "string") {
-      node[propName] = "";
+      setProperty(node, propName, "");
```

Comment:
@alexeygrinko-xo sorry i was not aware you are going to update setProperty method to set true value for boolean attributes when value is undefined or null. so here and on line 57 directly setting node[propName] is correct as it will remove boolean attributes

Reply:
You are right again :)